src/canvas/SelectableCanvas.ts

```diff
@@ -845,7 +846,10 @@ export class SelectableCanvas<EventSpec extends CanvasEvents = CanvasEvents>
     while (i--) {
       const target = objects[i];
       if (this._checkTarget(target, pointer)) {
-        if (isCollection(target) && target.subTargetCheck) {
+        if (
+          isCollection(target) &&
+          (target.subTargetCheck || target.interactive)
```

Comment:
target interactive requires subtargetcheck, so we shouldn't be checking for both.

Reply:
I thought I wrote about this in the description, sorry, updated now.

> Another change I have introduced is `interactive` including `subTargetCheck`
To able to select sub targets both need to be flagged as true.
`subTargetCheck` flag `searchPossibleTargets` to traverse the children of a group.
If `interactive` is flagged it means we want that to happen so essentially `interactive = true` contains `subTargetCheck = true`. it is more straight forward as well.